.github/workflows/ci.yml

```diff
@@ -299,6 +302,76 @@ jobs:
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
 
+  # description: Runs unit tests broken down by team and component
+  # test location: Operate
+  # owner: Core Features, Data Layers, Identity
+  consolidated-unit-tests:
+    if: needs.detect-changes.outputs.java-code-changes == 'true'
+    name: "[UT] ${{ matrix.componentName }} - ${{ matrix.suiteName }}"
+    needs: [ detect-changes ]
+    timeout-minutes: 10
+    permissions: { }  # GITHUB_TOKEN unused in this job
+    runs-on: gcp-perf-core-8-default
+    strategy:
+      fail-fast: false
+      matrix:
+        component: [ operate ]
+        suite: [ coreFeatures, dataLayer, identity ]
+        include:
+          - suite: coreFeatures
+            mavenArguments: "-DskipDataLayerUTs -DskipDataLayerUTs"
+            suiteName: Core Features
+          - suite: dataLayer
+            mavenArguments: "-DskipCoreFeaturesUTs -DskipIdentityUTs"
+            suiteName: Data Layers
+          - suite: identity
+            mavenArguments: "-DskipDataLayerUTs -DskipCoreFeaturesUTs"
+            suiteName: Identity
+          - component: operate
+            componentName: Operate
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-build
+        with:
+          maven-cache-key-modifier: consolidated-ut-${{ matrix.component }}-${{ matrix.suite }}
+          vault-address: ${{ secrets.VAULT_ADDR }}
+          vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
+          vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+      - name: Create build output log file
+        run: echo "BUILD_OUTPUT_FILE_PATH=$(mktemp)" >> $GITHUB_ENV
+      - name: Run Unit Tests
+        run: >
+          ./mvnw -f ${{matrix.component}} verify -T1C -B -P skipFrontendBuild
+          -DskipITs
+          ${{matrix.mavenArguments}}
+          -DskipDockerProfile
+          -DskipChecks
+          | tee "${BUILD_OUTPUT_FILE_PATH}"
+      - name: Analyze Test Runs
+        id: analyze-test-run
+        if: always()
+        uses: ./.github/actions/analyze-test-runs
+        with:
+          buildOutputFilePath: ${{ env.BUILD_OUTPUT_FILE_PATH }}
+      - name: Upload test artifacts
+        uses: ./.github/actions/collect-test-artifacts
+        if: ${{ failure() || cancelled() || steps.analyze-test-run.outputs.flakyTests != '' }}
+        with:
+          name: "[UT] ${{ matrix.component }} ${{ matrix.suite }}"
+      - name: Observe build status
+        if: always()
+        continue-on-error: true
+        uses: ./.github/actions/observe-build-status
+        with:
+          job_name: "consolidated-unit-tests/${{ matrix.component }}/${{ matrix.suite }}"
+          build_status: ${{ job.status }}
+          user_reason: ${{ (steps.analyze-test-run.outputs.flakyTests != '') && 'flaky-tests' || '' }}
+          user_description: ${{ matrix.suiteName }}
```

Comment:
💭 I'd go for something machine-readable and extensible like `user_description: "owner:${{ matrix.suite }}"`

(Can also be another noun like `suite:...` or `team:...`)

That later allows us to search this database column corresponding to `user_description` in a more structured way to identify a) whether owner information is present and if so b) parse the name out from it.